packages/tasks/src/model-libraries.ts

```diff
@@ -406,6 +406,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"mlc-chat-config.json"`,
 	},
+	moshi: {
+		prettyLabel: "Moshi",
+		repoName: "Moshi",
+		repoUrl: "https://github.com/kyutai-labs/moshi",
+		filter: false,
+		countDownloads: `path:"tokenizer-e351c8d8-checkpoint125.safetensors"`,
```

Comment:
this looks super specific no?

Reply:
Any thoughts about having

```suggestion
		countDownloads: `path:"model.safetensors" OR path:"model.q4.safetensors" OR path:"model.q8.safetensors" OR path:"model.gguf" OR path:"model.q4.gguf" OR path:"model.q8.gguf"`,
```
?

It's long and ugly but at least if a new tokenizer is pushed, it won't break the download count. And it wouldn't duplicate counts if I checked correctly.

Otherwise I'm fine with the current solution, there is no perfect one anyway.